.circleci/config.yml

```diff
@@ -222,6 +222,7 @@ jobs:
       - doc-deps-install
 
       - doc-build
+
```

Comment:
Unrelated?

Reply:
Yes, this was a naïve attempt to see if it affected the execution of the next steps...